addons/resource.language.en_gb/resources/strings.po

```diff
@@ -15371,13 +15374,13 @@ msgstr ""
 #. Description of settings category "Services -> Zeroconf" with label #1259
 #: system/settings/settings.xml
 msgctxt "#36341"
-msgid "Category containing settings for how the zeroconf network discovery service is handled, required for AirPlay."
+msgid "Category containing settings for how the \"Zeroconf\" network discovery service is handled, required for AirPlay."
```

Comment:
no need for quotes here as it's no cross reference but the description of the settings section itself when focused.